src/lib/logger.rs

```diff
@@ -133,15 +134,24 @@ pub(crate) fn init(options: &LoggerOptions) {
     let result = fern::Dispatch::new()
         .format(move |out, message, record| {
             let name = env!("CARGO_PKG_NAME");
+            let level = env::var("CARGO_MAKE_INTERNAL_RECURSION")
+                .unwrap_or_else(|_| "0".to_string());
+
             let record_level = record.level();
 
             let name_fmt = get_formatted_name(&name, color);
 
             let record_level_fmt = get_formatted_log_level(&record_level, color);
 
+            let mut log_prefix = format!("[{}", &name_fmt).to_string();
+            if level != "0" {
+                log_prefix.push_str(&format!(" [{}]", &level));
+            }
+            log_prefix.push_str("]");
+
             out.finish(format_args!(
-                "[{}] {} - {}",
-                &name_fmt, &record_level_fmt, &message
+                "{} {} - {}",
```

Comment:
i would change this one back to the original with added numeric one
"[{}]{} {} - {}",
but now that we added {} we can simply add a new binded variable to it that by default is empty string or any value you want like "[number]" or exmaple.
makes the format more understandable.
log prefix can stay the same and when building the formatted one, we can easily see what is in it.

Reply:
still prefer the change to fit my original formatting request.